### PR TITLE
GraphQL course 🎉

### PR DESCRIPTION
**Pull request by johnymontana**
This PR adds content for the updated GraphQL course featuring the Neo4j GraphQL Library and the Neo4j GraphQL Toolbox

Leaving this as DRAFT status for now, until the following are resolved:

- [x] resolve any remaining TODOs and FIXMEs (there are a handful)
- [x] update quiz questions / hints / solutions
- [x] use Sandbox secure bolt proxy endpoint
- [x] final proofread / content edit

### CODE REVIEW

**Review by adam-cowley — CHANGES_REQUESTED** [review]

File: asciidoc/courses/graphql-basics/course.adoc [hunk 1]
```diff
@@ -1,64 +1,47 @@
...
 :status: draft
-:usecase: blank-sandbox
+:usecase: recommendations
+:duration: 2 hours
+:key-points: GraphQL fundamentals, Using the Neo4j GraphQL Toolbox, Creating GraphQL APIs backed by Neo4j graph database
 
+// == Course Description
 
-== About the course
+Welcome to GraphAcademy and an Introduction to Neo4j and GraphQL.
```
[the remark this answers is not in the document]
Comment (martinohanlon) [reply asciidoc/courses/graphql-basics/course.adoc:11]: done

File: asciidoc/courses/graphql-basics/modules/1-setup/lessons/1-graphql/lesson.adoc [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+    credits
+    lessons() {
+      title
+      content
+    }
+  }
+}
+----
+
+> The selection set and can be thought of as a traversal through the data graph.
```
[the remark this answers is not in the document]
Comment (martinohanlon) [reply asciidoc/courses/graphql-basics/modules/1-setup/lessons/1-graphql/lesson.adoc:157]: done

File: asciidoc/courses/graphql-basics/modules/1-setup/lessons/1-graphql/lesson.adoc [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+  }
+}
+----
+
+=== Benefits of GraphQL
+
+Some of the benefits of GraphQL include:
+
+* **Overfetching** - sending less data over the wire
+* **Underfetching** - everything the client needs in a single request
```
[the remark this answers is not in the document]
Comment (martinohanlon) [reply asciidoc/courses/graphql-basics/modules/1-setup/lessons/1-graphql/lesson.adoc:207]: simplified the list of benefits

File: asciidoc/courses/graphql-basics/modules/1-setup/lessons/1-graphql/questions/nplus1.adoc [hunk 4]
```diff
@@ -0,0 +1,21 @@
+[.question]
+= 2. n+1 Query Problem
+
+The n+1 query problem is a common problem developers face when building GraphQL APIs and can lead to poor performance if not properly addressed.
+
+Is this statement true or false?
```
[the remark this answers is not in the document]
Comment (martinohanlon) [reply asciidoc/courses/graphql-basics/modules/1-setup/lessons/1-graphql/questions/nplus1.adoc:6]: simplified

File: asciidoc/courses/graphql-basics/modules/1-setup/lessons/2-neo4j-graphql/lesson.adoc [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+image::images/02graph_ql_toolbox.png[Screenshot of the GraphQL toolbox,width=500,align=center]
+
+include::{shared}/courses/apps/sandbox.adoc[tag="info"]
+
+Run this cypher query in the sandbox to see `Movie` nodes stored in the sandbox.
+
+[source,cypher]
+----
+MATCH (m:Movie) RETURN m
+----
```
[the remark this answers is not in the document]
Comment (martinohanlon) [reply asciidoc/courses/graphql-basics/modules/1-setup/lessons/2-neo4j-graphql/lesson.adoc:17]: done

File: asciidoc/courses/graphql-basics/modules/2-queries/lessons/1-graphql-queries/lesson.adoc [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+
+include::questions/question-2.adoc[leveloffset=+1]
+
+include::questions/question-3.adoc[leveloffset=+1]
+
+
+[.summary]
+== Summary
+
+In this lesson, you were introduced the Query API of the Neo4j GraphQL Library and the Neo4j GraphQL Toolbox. In the next lesson you will see how to build a more complex data graph using the GraphQL schema type definitions and the Neo4j GraphQL Toolbox.
```
[the remark this answers is not in the document]
Comment (martinohanlon) [reply asciidoc/courses/graphql-basics/modules/2-queries/lessons/1-graphql-queries/lesson.adoc:164]: 2 paragraphs is easier to read. done

File: asciidoc/courses/graphql-basics/modules/1-setup/lessons/3-neo4j-graphql-library/lesson.adoc [hunk 7]
```diff
@@ -0,0 +1,77 @@
+= The Neo4j GraphQL Library
+:order: 3
+
+The toolbox provides an easy to use interface over the https://neo4j.com/docs/graphql-manual/current/[Neo4j GraphQL Library ^].
+
+The fundamental goal of the Neo4j GraphQL Library is to make it easier to build GraphQL APIs backed by Neo4j.
+
+> It's important to point out that GraphQL is an API query language and NOT a database query language. The Neo4j GraphQL Library helps you build an API layer that sits between the client and database, not to execute GraphQL queries directly against the database.
```
[the remark this answers is not in the document]
Comment (martinohanlon) [reply asciidoc/courses/graphql-basics/modules/1-setup/lessons/3-neo4j-graphql-library/lesson.adoc:8]: done

File: asciidoc/courses/graphql-basics/course.adoc [hunk 8]
```diff
@@ -1,14 +1,11 @@
 = Introduction to Neo4j & GraphQL
 :categories: developer:2, graphql:1
 :caption: Learn how to create GraphQL API using Neo4j GraphQL Toolbox and Library.
-:status: draft
+:status: active
```
Comment (adam-cowley) [inline asciidoc/courses/graphql-basics/course.adoc:4]: If we merge with draft status, Neo4j employees will be able to enrol.

**Review by adam-cowley — APPROVED** [review]

File: asciidoc/courses/graphql-basics/course.adoc (hunk 8 above)
Comment (martinohanlon) [reply asciidoc/courses/graphql-basics/course.adoc]: Yes

I was having trouble in my dev environment getting the course to be open to enrollment, I change this in an attempt to get it to open.

Changed back to draft.

**Pull request merged**